R/oauth-token.r

```diff
@@ -197,10 +197,12 @@ oauth2.0_token <- function(endpoint, app, scope = NULL, user_params = NULL,
                            type = NULL, use_oob = getOption("httr_oob_default"),
                            as_header = TRUE,
                            use_basic_auth = FALSE,
-                           cache = getOption("httr_oauth_cache")) {
+                           cache = getOption("httr_oauth_cache"),
+                           config = list()) {
```

Comment:
I think this needs a more specific name because there are other places in the OAuth dance that could have config params. Maybe `config_init`?